conf/nginx.conf

```diff
@@ -2,22 +2,22 @@
 location __PATH__/ {
   proxy_pass http://127.0.0.1:__PORT_API_GATEWAY__/;
   proxy_http_version 1.1;
-  proxy_set_header Host $http_host;
+  proxy_set_header Host $host;
   proxy_set_header Upgrade $http_upgrade;
   proxy_set_header Connection $connection_upgrade;
   proxy_set_header X-Forwarded-Proto $scheme;
   proxy_buffering off;
 }
 
-location = __PATH__/ {
-  default_type text/plain;
-  return 200 "This is where Standard Notes - Syncing Server is installed.";
-}
+# location = __PATH__/ {
+#   default_type text/plain;
+#   return 200 "This is where Standard Notes - Syncing Server is installed.";
+# }
```

Comment:
Well uh this seemed important to have the test pass ?

Reply:
Yes, uh, can’t remember why i commented this out.